environments/production/common/versions.tf

```diff
@@ -16,6 +16,11 @@ terraform {
       source  = "hashicorp/archive"
       version = ">= 2.0.0"
     }
+
+    external = {
```

Comment:
Not strictly necessary but one less change for when we get the lambdas running in production